lib/registration.pm

```diff
@@ -369,14 +369,16 @@ sub _yast_scc_addons_handler {
 
 sub skip_package_hub_if_necessary {
     my ($addon) = @_;
+    my $skip_package_hub = 0;
     if (is_sle('15-SP2+') && $addon eq 'phub') {
         if (check_var('FLAVOR', 'Online')) {
             record_soft_failure('bsc#1151373 - Missing or broken repository after registering module PackageHub');
         } elsif (check_var('FLAVOR', 'Full')) {
             record_info('Full media: no PHUB', 'Skipping Package Hub, it is not available on offline scenarios - bsc#1157659');
         }
-        next;
+        $skip_package_hub = 1;
```

Comment:
We could just `return 1` and `return 0` without introducing variable. WDYT?

Reply:
My best practice says that only one return at the end of a function should be used.
But, I can change it.

Reply:
Sure, that's more matter of taste and won't use much more memory, so let's merge.